src/server/routes/index.js

```diff
@@ -3,14 +3,14 @@ const router = express.Router();
 const pc = require('./pathway-commons-router');
 const summary = require('./summary');
 
-
+router.use('/api/search', require('./search'));
```

Comment:
search is currently organized under /api/pc if we want /api/search then we should just scrap /api/pc

Reply:
Eventually, I was thinking we should remove all other 'search' (`/pc`, `/summary`) endpoints and just use the modules directly within this one. Unless there is a reason to expose those?

Reply:
Yeah that makes sense